synfig-core/src/synfig/listimporter.cpp

```diff
@@ -178,8 +178,9 @@ ListImporter::get_sub_importer(const RendDesc &renddesc, Time time, ProgressCall
 	Importer::Handle importer(Importer::open(FileSystem::Identifier(FileSystemNative::instance(), filename)));
 	if(!importer)
 	{
-		if(cb)cb->error(_("Unable to open ")+filename);
-		else synfig::error(_("Unable to open ")+filename);
+		std::string error_msg = strprintf(_("Unable to open %s"), filename.c_str());
```

Comment:
Is that correct? I mean, maybe there should be `u8_str()`?

Reply:
`filename` here is a `synfig::String` variable, not `synfig::filesystem::Path`. So, it doesn't have the `u8_str()` method.
I don't know why I submitted this change XD